### PR TITLE
Update generated reference for kubeadm

### PR DESCRIPTION
**Pull request by tengqm**
This PR updates the generated kubeadm reference so that we will put, for example, `<comma-separated 'key=value' pairs>` rather than `mapStringString` for flag types.

### CODE REVIEW

**Comment by celestehorgan** [comment]
@kubernetes/sig-cluster-lifecycle – could you please take a look at this PR and provide an LGTM from a SIG/technical perspective?

**Comment by neolit123** [comment]
> kubernetes/sig-cluster-lifecycle – could you please take a look at this PR and provide an LGTM from a SIG/technical perspective?

hi, i added LGTM above. the change seems fine.

File: content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_kubeconfig_user.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+to generate the reference documentation, please read
+[Contributing to the reference documentation](/docs/contribute/generate-ref-docs/).
+To update the reference conent, please follow the 
+[Contributing upstream](/docs/contribute/generate-ref-docs/contribute-upstream/)
+guide. You can file document formatting bugs against the
+[reference-docs](https://github.com/kubernetes-sigs/reference-docs/) project.
+-->
+
+
+Output a kubeconfig file for an additional user
```
Comment (neolit123) [inline content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_kubeconfig_user.md:13]: taking a look again here, the kubeadm_alpha_kubeconfig command is deprecated.
prior art indicated that the generator skips Cobra commands that have the Deprecated field, so not sure why it's included here again. :thinking: 

i expected kubeadm_alpha to be empty for 1.21.

not a blocker, but just wanted to mention this.

**Comment by tengqm** [comment]
/close
Another PR has updated the reference.

**Pull request closed without merging**